### PR TITLE
Fix missing filename when specified JSON file for the "--globals" argument cannot be parsed to a dict

### PR DESCRIPTION
**Pull request by j3rrywan9**
Fix https://github.com/confluentinc/ducktape/issues/273

### CODE REVIEW

**Comment by j3rrywan9** [comment]
[clabot:check]

File: tests/command_line/check_main.py [hunk 1]
```diff
@@ -155,3 +155,20 @@ def check_non_dict(self):
...
+
+    def check_non_dict_from_file(self):
+        """Validate behavior when given file containing valid JSON which does not parse as a dict"""
+        _, fname = tempfile.mkstemp()
+        try:
+            with open(fname, "w") as fh:
+                # Write valid JSON which does not parse as a dict
+                fh.write(valid_json_not_dict)
+
+            with pytest.raises(ValueError) as excinfo:
```
Comment (stan-confluent) [inline tests/command_line/check_main.py:168]: Nit: you can use `pytest.raises(ValueError, matches=fname):` also.

**Review by stan-confluent — APPROVED** [review]
lgtm, thanks for fixing this!

**Review by stan-confluent — APPROVED** [review]

**Comment by stan-confluent** [comment]
@j3rrywan9 Thanks for updating the test. Style command failed, but I don't think this is your change, so I'm going to merge this into 0.8.x and then push it up to master.
Thanks!

**Pull request merged**